Sensotrend uploader 2.34.0

jlaunonen opened the pull request.

gniezen committed:
remove test

gniezen committed:
Merge pull request #1298 from tidepool-org/snyk-upgrade-83d6d3a211e9f46a5513f1143ec3a78e

[Snyk] Upgrade concurrently from 5.2.0 to 5.3.0

gniezen committed:
Merge branch 'master' into snyk-upgrade-cfeb9a62b28dd8c3061af244eb6dad39

gniezen committed:
Merge pull request #1292 from tidepool-org/snyk-upgrade-cfeb9a62b28dd8c3061af244eb6dad39

[Snyk] Upgrade react-redux from 7.2.0 to 7.2.1

gniezen committed:
Merge branch 'master' into snyk-upgrade-41577bb938f897943fc5338488c73d34

gniezen committed:
Merge pull request #1297 from tidepool-org/snyk-upgrade-41577bb938f897943fc5338488c73d34

[Snyk] Upgrade serialport from 9.0.0 to 9.0.1

snyk-bot committed:
fix: upgrade open from 7.1.0 to 7.2.0

Snyk has created this PR to upgrade open from 7.1.0 to 7.2.0.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade mini-css-extract-plugin from 0.9.0 to 0.10.0

Snyk has created this PR to upgrade mini-css-extract-plugin from 0.9.0 to 0.10.0.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

gniezen committed:
Merge branch 'master' into s3-test

gniezen committed:
use cli override for only publishing to github

gniezen committed:
v2.33.0-s3-test.9

gniezen committed:
rename publish-s3 to just publish

gniezen committed:
switch params around

gniezen committed:
remove github override as test

gniezen committed:
put c.publish back

gniezen committed:
try using c.publish.provider instead

gniezen committed:
trying it the other way around again

gniezen committed:
Merge pull request #1303 from tidepool-org/snyk-upgrade-4a57e11ae4c5e515450c731cdc048e8d

[Snyk] Upgrade mini-css-extract-plugin from 0.9.0 to 0.10.0

gniezen committed:
Merge branch 'master' into snyk-upgrade-cd9506ce7f8fc2a0ba07782e451b1ef0

gniezen committed:
try different config format

gniezen committed:
fix typo

gniezen committed:
flip providers around to see if that merges with cli config

gniezen committed:
Merge pull request #1301 from tidepool-org/snyk-upgrade-cd9506ce7f8fc2a0ba07782e451b1ef0

[Snyk] Upgrade open from 7.1.0 to 7.2.0

gniezen committed:
attempt to change publish path

gniezen committed:
trying some more cli config options

snyk-bot committed:
fix: upgrade rollbar from 2.19.2 to 2.19.3

Snyk has created this PR to upgrade rollbar from 2.19.2 to 2.19.3.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade serialport from 9.0.0 to 9.0.1

Snyk has created this PR to upgrade serialport from 9.0.0 to 9.0.1.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/ae26ef2f-f4cf-48b7-987b-8ea2917d443a?utm_source=github&utm_medium=upgrade-pr

gniezen committed:
check to see if custom config is merged with rest

gniezen committed:
whole config for build

gniezen committed:
Merge pull request #1305 from tidepool-org/snyk-upgrade-09367bccbdb147ba066d0870f031e5cc

[Snyk] Upgrade serialport from 9.0.0 to 9.0.1

gniezen committed:
Merge branch 'master' into snyk-upgrade-03c902947c570bc605b9cea67fdf5524

gniezen committed:
Merge branch 'master' into contour-plus

gniezen committed:
Merge pull request #1304 from tidepool-org/snyk-upgrade-03c902947c570bc605b9cea67fdf5524

[Snyk] Upgrade rollbar from 2.19.2 to 2.19.3

gniezen committed:
Merge branch 'master' into s3-test

gniezen committed:
put package and publish in the right places

gniezen committed:
v2.33.0-s3-test.10

gniezen committed:
Merge branch 's3-test' of github.com:tidepool-org/uploader into s3-test

gniezen committed:
changing publish to publish-s3 as we're probably conflicting with publishing to npm

gniezen committed:
v2.33.0-s3-test.11

snyk-bot committed:
fix: upgrade mini-css-extract-plugin from 0.10.0 to 0.11.0

Snyk has created this PR to upgrade mini-css-extract-plugin from 0.10.0 to 0.11.0.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade open from 7.2.0 to 7.2.1

Snyk has created this PR to upgrade open from 7.2.0 to 7.2.1.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

jlaunonen committed:
Add localization for Finnish.

jlaunonen committed:
Add few more device list strings to localization.

jehernandezrodriguez committed:
Merge with master

jehernandezrodriguez committed:
Handle i18next in test

jehernandezrodriguez committed:
Adding translations of errors

jlaunonen committed:
Localize times in UI.

jlaunonen committed:
Add localization for upload time range label.

gniezen committed:
prevent layout issues with long device names

gniezen committed:
Merge branch 'master' into contour-plus

gniezen committed:
v.33.0-contour-plus.2

gniezen committed:
Merge pull request #1310 from tidepool-org/snyk-upgrade-31247b264a72b546212b53d648435162

[Snyk] Upgrade open from 7.2.0 to 7.2.1

gniezen committed:
Merge branch 'master' into snyk-upgrade-557bdac8809cbfa167f517c2e2a08edb

snyk-bot committed:
fix: upgrade eslint-plugin-react from 7.20.6 to 7.21.3

Snyk has created this PR to upgrade eslint-plugin-react from 7.20.6 to 7.21.3.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

gniezen committed:
Merge pull request #1309 from tidepool-org/snyk-upgrade-557bdac8809cbfa167f517c2e2a08edb

[Snyk] Upgrade mini-css-extract-plugin from 0.10.0 to 0.11.0

gniezen committed:
Merge branch 'master' into snyk-upgrade-c22c0d1b95597b8af3e4358589231896

gniezen committed:
Merge pull request #1316 from tidepool-org/snyk-upgrade-c22c0d1b95597b8af3e4358589231896

[Snyk] Upgrade eslint-plugin-react from 7.20.6 to 7.21.3

gniezen committed:
Merge branch 'master' into snyk-upgrade-7bcc5bdad5b04cf72cd907827528e9cd

gniezen committed:
Merge branch 'master' into s3-test

gniezen committed:
move electron-builder config into .js file

gniezen committed:
also add actual config file

gniezen committed:
v2.33.0-s3-test.12

gniezen committed:
look for 'true' instead

gniezen committed:
2.33.0-s3-test.13

gniezen committed:
increase time between individual send requests, to not starve CIQ process on pump

gniezen committed:
Merge branch 'master' into tandem-fixes

gniezen committed:
v2.33.0-tandem-fixes.2

snyk-bot committed:
fix: upgrade mini-css-extract-plugin from 0.11.0 to 0.12.0

Snyk has created this PR to upgrade mini-css-extract-plugin from 0.11.0 to 0.12.0.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade rollbar-sourcemap-webpack-plugin from 2.11.4 to 2.12.0

Snyk has created this PR to upgrade rollbar-sourcemap-webpack-plugin from 2.11.4 to 2.12.0.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

gniezen committed:
specify github release type when tagged

gniezen committed:
v2.33.0-s3-test.14

gniezen committed:
fix publish object; check if env var exists

gniezen committed:
v2.33.0-s3-test.15

gniezen committed:
add github owner

gniezen committed:
v2.33.0-s3-test.16

gniezen committed:
re-enable whitelisting

gniezen committed:
change visual element so that QA can verify published binaries didn't change

gniezen committed:
change visual element back after successful test; remove test suffix

gniezen committed:
Merge pull request #1318 from tidepool-org/snyk-upgrade-1a46517568b6e363ff59027ca729f0fc

[Snyk] Upgrade rollbar-sourcemap-webpack-plugin from 2.11.4 to 2.12.0

gniezen committed:
Merge branch 'master' into snyk-upgrade-9dc8cf6988797e8531d564454c724f32

gniezen committed:
Merge pull request #1317 from tidepool-org/snyk-upgrade-9dc8cf6988797e8531d564454c724f32

[Snyk] Upgrade mini-css-extract-plugin from 0.11.0 to 0.12.0

dependabot[bot] committed:
build(deps-dev): bump electron from 8.5.0 to 8.5.2

Bumps [electron](https://github.com/electron/electron) from 8.5.0 to 8.5.2.
- [Release notes](https://github.com/electron/electron/releases)
- [Changelog](https://github.com/electron/electron/blob/master/docs/breaking-changes.md)
- [Commits](https://github.com/electron/electron/compare/v8.5.0...v8.5.2)

Signed-off-by: dependabot[bot] <support@github.com>

gniezen committed:
Merge branch 'master' into contour-plus

gniezen committed:
remove test suffix

gniezen committed:
Merge branch 'master' into power-only

gniezen committed:
remove test suffix

gniezen committed:
Merge branch 'master' into snyk-upgrade-7bcc5bdad5b04cf72cd907827528e9cd

gniezen committed:
Merge pull request #1236 from tidepool-org/snyk-upgrade-7bcc5bdad5b04cf72cd907827528e9cd

[Snyk] Upgrade tidepool-platform-client from 0.42.0 to 0.43.0 (UPLOAD-374)

gniezen committed:
Merge branch 'master' into dependabot/npm_and_yarn/electron-8.5.2

dependabot[bot] committed:
build(deps): bump http-proxy from 1.18.0 to 1.18.1

Bumps [http-proxy](https://github.com/http-party/node-http-proxy) from 1.18.0 to 1.18.1.
- [Release notes](https://github.com/http-party/node-http-proxy/releases)
- [Changelog](https://github.com/http-party/node-http-proxy/blob/master/CHANGELOG.md)
- [Commits](https://github.com/http-party/node-http-proxy/compare/1.18.0...1.18.1)

Signed-off-by: dependabot[bot] <support@github.com>

gniezen committed:
Merge pull request #1315 from tidepool-org/dependabot/npm_and_yarn/electron-8.5.2

build(deps-dev): bump electron from 8.5.0 to 8.5.2

gniezen committed:
Merge branch 'master' into dependabot/npm_and_yarn/http-proxy-1.18.1

gniezen committed:
Merge pull request #1311 from tidepool-org/dependabot/npm_and_yarn/http-proxy-1.18.1

build(deps): bump http-proxy from 1.18.0 to 1.18.1

gniezen committed:
Merge branch 'master' into s3-test

gniezen committed:
Merge pull request #1276 from tidepool-org/s3-test

Only publish to S3 on tagged release (UPLOAD-376)

snyk-bot committed:
fix: upgrade tidepool-platform-client from 0.43.0 to 0.44.0

Snyk has created this PR to upgrade tidepool-platform-client from 0.43.0 to 0.44.0.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade multiple dependencies with Snyk

Snyk has created this PR to upgrade:
  - @babel/core from 7.11.4 to 7.12.0.
    See this package in npm: 
  - @babel/plugin-proposal-export-namespace-from from 7.10.4 to 7.12.0.
    See this package in npm: 
  - @babel/plugin-proposal-function-bind from 7.10.5 to 7.11.5.
    See this package in npm: 
  - @babel/plugin-proposal-logical-assignment-operators from 7.11.0 to 7.12.0.
    See this package in npm: 
  - @babel/plugin-proposal-nullish-coalescing-operator from 7.10.4 to 7.12.0.
    See this package in npm: 
  - @babel/plugin-proposal-numeric-separator from 7.10.4 to 7.12.0.
    See this package in npm: 
  - @babel/plugin-proposal-optional-chaining from 7.11.0 to 7.12.0.
    See this package in npm: 
  - @babel/polyfill from 7.10.4 to 7.11.5.
    See this package in npm: 
  - @babel/preset-env from 7.11.0 to 7.12.0.
    See this package in npm: 
  - @babel/register from 7.10.5 to 7.12.0.
    See this package in npm: 

See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade electron-builder from 22.8.0 to 22.9.0

Snyk has created this PR to upgrade electron-builder from 22.8.0 to 22.9.0.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade sinon from 9.0.3 to 9.2.0

Snyk has created this PR to upgrade sinon from 9.0.3 to 9.2.0.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

gniezen committed:
Merge branch 'master' into contour-plus

gniezen committed:
Merge pull request #1307 from tidepool-org/contour-plus

Add support for Contour Plus meter (UPLOAD-365)

gniezen committed:
Merge branch 'master' into power-only

gniezen committed:
Merge pull request #1275 from tidepool-org/power-only

add error messaging re power-only USB cables (UPLOAD-382)

gniezen committed:
Merge branch 'master' into snyk-upgrade-9f0b9d4f44a80d361eb468bdfe07c76c

gniezen committed:
Merge pull request #1322 from tidepool-org/snyk-upgrade-9f0b9d4f44a80d361eb468bdfe07c76c

[Snyk] Upgrade sinon from 9.0.3 to 9.2.0

gniezen committed:
Merge branch 'master' into snyk-upgrade-a1492c30280d7571f9b3314aee56f336

gniezen committed:
bump electron-builder to v22.9.1

gniezen committed:
Merge pull request #1321 from tidepool-org/snyk-upgrade-a1492c30280d7571f9b3314aee56f336

[Snyk] Upgrade electron-builder from 22.8.0 to 22.9.0

gniezen committed:
Merge branch 'master' into snyk-upgrade-6082320173fc55e16b6141710d00e190

snyk-bot committed:
fix: upgrade @babel/runtime-corejs2 from 7.11.2 to 7.12.1

Snyk has created this PR to upgrade @babel/runtime-corejs2 from 7.11.2 to 7.12.1.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

mrinnetmaki committed:
Use connect.sensotrend.fi as the default server

sensotrend-ci committed:
Use Finnish as the default language

jehernandezrodriguez committed:
Merge with master

jehernandezrodriguez committed:
Include files of locales in builder file

gniezen committed:
Merge branch 'master' into tandem-fixes

gniezen committed:
remove test suffix

gniezen committed:
Merge pull request #1308 from tidepool-org/tandem-fixes

Fix for Tandem uploads stalling (UPLOAD-290)

gniezen committed:
v2.34.0

gniezen committed:
Merge pull request #1324 from tidepool-org/v2.34.0

v2.34.0

gniezen committed:
Merge branch 'master' into snyk-upgrade-d184eb88928a81b4bd7fa7ca1dbaaa8e

gniezen committed:
Merge pull request #1323 from tidepool-org/snyk-upgrade-d184eb88928a81b4bd7fa7ca1dbaaa8e

[Snyk] Upgrade @babel/runtime-corejs2 from 7.11.2 to 7.12.1

gniezen committed:
Merge branch 'master' into snyk-upgrade-6082320173fc55e16b6141710d00e190

gniezen committed:
Merge pull request #1320 from tidepool-org/snyk-upgrade-6082320173fc55e16b6141710d00e190

[Snyk] Upgrade: @babel/core, @babel/plugin-proposal-export-namespace-from, @babel/plugin-proposal-function-bind, @babel/plugin-proposal-logical-assignment-operators, @babel/plugin-proposal-nullish-coalescing-operator, @babel/plugin-proposal-numeric-separator, @babel/plugin-proposal-optional-chaining, @babel/polyfill, @babel/preset-env, @babel/register

gniezen committed:
Merge branch 'master' into snyk-upgrade-a6945be3f68c5c8246d5498f6cfd118b

gniezen committed:
Merge pull request #1319 from tidepool-org/snyk-upgrade-a6945be3f68c5c8246d5498f6cfd118b

[Snyk] Upgrade tidepool-platform-client from 0.43.0 to 0.44.0

snyk-bot committed:
fix: upgrade eslint-plugin-react from 7.21.3 to 7.21.5

Snyk has created this PR to upgrade eslint-plugin-react from 7.21.3 to 7.21.5.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade multiple dependencies with Snyk

Snyk has created this PR to upgrade:
  - @babel/core from 7.12.0 to 7.12.3.
    See this package in npm: 
  - @babel/plugin-proposal-class-properties from 7.10.4 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-proposal-decorators from 7.10.5 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-proposal-do-expressions from 7.10.4 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-proposal-export-default-from from 7.10.4 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-proposal-export-namespace-from from 7.12.0 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-proposal-function-bind from 7.11.5 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-proposal-function-sent from 7.10.4 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-proposal-json-strings from 7.10.4 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-proposal-logical-assignment-operators from 7.12.0 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-proposal-nullish-coalescing-operator from 7.12.0 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-proposal-numeric-separator from 7.12.0 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-proposal-optional-chaining from 7.12.0 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-proposal-pipeline-operator from 7.10.5 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-proposal-throw-expressions from 7.10.4 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-transform-classes from 7.10.4 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-transform-react-constant-elements from 7.10.4 to 7.12.1.
    See this package in npm: 
  - @babel/plugin-transform-react-inline-elements from 7.10.4 to 7.12.1.
    See this package in npm: 
  - @babel/polyfill from 7.11.5 to 7.12.1.
    See this package in npm: 
  - @babel/preset-env from 7.12.0 to 7.12.1.
    See this package in npm: 
  - @babel/preset-react from 7.10.4 to 7.12.1.
    See this package in npm: 
  - @babel/register from 7.12.0 to 7.12.1.
    See this package in npm: 

See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade open from 7.2.1 to 7.3.0

Snyk has created this PR to upgrade open from 7.2.1 to 7.3.0.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade react-hot-loader from 4.12.21 to 4.13.0

Snyk has created this PR to upgrade react-hot-loader from 4.12.21 to 4.13.0.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade rollbar from 2.19.3 to 2.19.4

Snyk has created this PR to upgrade rollbar from 2.19.3 to 2.19.4.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

gniezen committed:
Merge pull request #1329 from tidepool-org/snyk-upgrade-22169eef4523b64f5d7dc09992d7654d

[Snyk] Upgrade rollbar from 2.19.3 to 2.19.4

gniezen committed:
Merge branch 'master' into snyk-upgrade-81d7d74ffb6d8cff1c1db25b26691969

gniezen committed:
Merge branch 'master' into webmtp

gniezen committed:
Merge branch 'master' into webmtp

gniezen committed:
Merge pull request #1328 from tidepool-org/snyk-upgrade-81d7d74ffb6d8cff1c1db25b26691969

[Snyk] Upgrade react-hot-loader from 4.12.21 to 4.13.0

gniezen committed:
Merge branch 'master' into snyk-upgrade-e9fd1cd28e1135bb7375bec2b01e3cc5

gniezen committed:
Merge pull request #1327 from tidepool-org/snyk-upgrade-e9fd1cd28e1135bb7375bec2b01e3cc5

[Snyk] Upgrade open from 7.2.1 to 7.3.0

gniezen committed:
Merge branch 'master' into snyk-upgrade-198dcb2675c36d3614874f85f0a502bf

gniezen committed:
Merge pull request #1326 from tidepool-org/snyk-upgrade-198dcb2675c36d3614874f85f0a502bf

[Snyk] Upgrade: @babel/core, @babel/plugin-proposal-class-properties, @babel/plugin-proposal-decorators, @babel/plugin-proposal-do-expressions, @babel/plugin-proposal-export-default-from, @babel/plugin-proposal-export-namespace-from, @babel/plugin-proposal-function-bind, @babel/plugin-proposal-function-sent, @babel/plugin-proposal-json-strings, @babel/plugin-proposal-logical-assignment-operators, @babel/plugin-proposal-nullish-coalescing-operator, @babel/plugin-proposal-numeric-separator, @babel/plugin-proposal-optional-chaining, @babel/plugin-proposal-pipeline-operator, @babel/plugin-proposal-throw-expressions, @babel/plugin-transform-classes, @babel/plugin-transform-react-constant-elements, @babel/plugin-transform-react-inline-elements, @babel/polyfill, @babel/preset-env, @babel/preset-react, @babel/register

gniezen committed:
Merge branch 'master' into snyk-upgrade-09f6e64cbd638f4d693619d0867ae3d9

gniezen committed:
Merge pull request #1325 from tidepool-org/snyk-upgrade-09f6e64cbd638f4d693619d0867ae3d9

[Snyk] Upgrade eslint-plugin-react from 7.21.3 to 7.21.5

snyk-bot committed:
fix: upgrade @hot-loader/react-dom from 16.13.0 to 16.14.0

Snyk has created this PR to upgrade @hot-loader/react-dom from 16.13.0 to 16.14.0.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade style-loader from 1.2.1 to 1.3.0

Snyk has created this PR to upgrade style-loader from 1.2.1 to 1.3.0.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade optimize-css-assets-webpack-plugin from 5.0.3 to 5.0.4

Snyk has created this PR to upgrade optimize-css-assets-webpack-plugin from 5.0.3 to 5.0.4.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

gniezen committed:
Merge pull request #1332 from tidepool-org/snyk-upgrade-e09ba18b5be057ed4467937021aa9c0f

[Snyk] Upgrade style-loader from 1.2.1 to 1.3.0

gniezen committed:
Merge branch 'master' into snyk-upgrade-1626ad9a5cada579cf1fbcd258768ded

gniezen committed:
Merge pull request #1333 from tidepool-org/snyk-upgrade-1626ad9a5cada579cf1fbcd258768ded

[Snyk] Upgrade optimize-css-assets-webpack-plugin from 5.0.3 to 5.0.4

gniezen committed:
Merge branch 'master' into snyk-upgrade-4cc32e5f4111898e1300fd834bbb53b0

gniezen committed:
Merge pull request #1331 from tidepool-org/snyk-upgrade-4cc32e5f4111898e1300fd834bbb53b0

[Snyk] Upgrade @hot-loader/react-dom from 16.13.0 to 16.14.0

snyk-bot committed:
fix: upgrade eslint-plugin-import from 2.22.0 to 2.22.1

Snyk has created this PR to upgrade eslint-plugin-import from 2.22.0 to 2.22.1.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade webpack-bundle-analyzer from 3.8.0 to 3.9.0

Snyk has created this PR to upgrade webpack-bundle-analyzer from 3.8.0 to 3.9.0.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade webpack from 4.44.1 to 4.44.2

Snyk has created this PR to upgrade webpack from 4.44.1 to 4.44.2.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

jlaunonen committed:
Merge remote-tracking branch 'tidepool/master' into main

- Move build-related customization to electron-builder-publish.js.

jlaunonen committed:
Add publish configuration for all package variants.

Otherwise only default configuration is used, producing broken packages.

gniezen committed:
Make difference between class and instantiation clearer

Co-authored-by: Lennart Goedhart <lennart@tidepool.org>

gniezen committed:
constructor should start with a capital letter

gniezen committed:
Merge branch 'master' into webmtp

gniezen committed:
v2.34.0-webmtp.4

gniezen committed:
Merge pull request #1336 from tidepool-org/snyk-upgrade-9d68877f7a50e9ed3aa37390e91d3fe1

[Snyk] Upgrade webpack from 4.44.1 to 4.44.2

gniezen committed:
Merge branch 'master' into snyk-upgrade-7a3df9ce1055b10dc9f1df3e6a98f455

gniezen committed:
Merge pull request #1335 from tidepool-org/snyk-upgrade-7a3df9ce1055b10dc9f1df3e6a98f455

[Snyk] Upgrade webpack-bundle-analyzer from 3.8.0 to 3.9.0

snyk-bot committed:
fix: upgrade serialport from 9.0.1 to 9.0.2

Snyk has created this PR to upgrade serialport from 9.0.1 to 9.0.2.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

gniezen committed:
Merge branch 'master' into webmtp

gniezen committed:
Merge pull request #1337 from tidepool-org/snyk-upgrade-5625239392f303069f31c902081cff85

[Snyk] Upgrade serialport from 9.0.1 to 9.0.2

gniezen committed:
Merge branch 'master' into snyk-upgrade-c9e6046f94f52773a6c962aa9670a87b

gniezen committed:
Merge pull request #1334 from tidepool-org/snyk-upgrade-c9e6046f94f52773a6c962aa9670a87b

[Snyk] Upgrade eslint-plugin-import from 2.22.0 to 2.22.1

Commit:
Merge branch 'master' into webmtp

snyk-bot committed:
fix: upgrade eslint-plugin-jsx-a11y from 6.3.1 to 6.4.1

Snyk has created this PR to upgrade eslint-plugin-jsx-a11y from 6.3.1 to 6.4.1.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade react-redux from 7.2.1 to 7.2.2

Snyk has created this PR to upgrade react-redux from 7.2.1 to 7.2.2.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

gniezen committed:
Merge pull request #1338 from tidepool-org/snyk-upgrade-a346bd000e2da6eb3dd5ddc81b1db8f0

[Snyk] Upgrade eslint-plugin-jsx-a11y from 6.3.1 to 6.4.1

gniezen committed:
Merge branch 'master' into snyk-upgrade-b665cd71bcbb1f243525fe095645c69f

gniezen committed:
Merge pull request #1339 from tidepool-org/snyk-upgrade-b665cd71bcbb1f243525fe095645c69f

[Snyk] Upgrade react-redux from 7.2.1 to 7.2.2

snyk-bot committed:
fix: upgrade multiple dependencies with Snyk

Snyk has created this PR to upgrade:
  - react from 16.13.1 to 16.14.0.
    See this package in npm: 
  - react-dom from 16.13.1 to 16.14.0.
    See this package in npm: 

See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade serialport from 9.0.1 to 9.0.2

Snyk has created this PR to upgrade serialport from 9.0.1 to 9.0.2.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/ae26ef2f-f4cf-48b7-987b-8ea2917d443a?utm_source=github&utm_medium=upgrade-pr

gniezen committed:
Merge pull request #1341 from tidepool-org/snyk-upgrade-727d9234a8a78d1c740725048c7e4c71

[Snyk] Upgrade serialport from 9.0.1 to 9.0.2

gniezen committed:
Merge branch 'master' into snyk-upgrade-11800484ae0e2ca28c44a8572ff1506e

gniezen committed:
Merge pull request #1340 from tidepool-org/snyk-upgrade-11800484ae0e2ca28c44a8572ff1506e

[Snyk] Upgrade: react, react-dom

jlaunonen committed:
Merge remote-tracking branch 'jehernandezrodriguez/multi-languages' into main

jlaunonen committed:
Merge branch 'localization' into main

jlaunonen committed:
Merge branch 'main' into sensotrend-uploader

jlaunonen committed:
Customize few Tidepool link references to Sensotrend.

jlaunonen committed:
Localize Windows upgrade prompt.

snyk-bot committed:
fix: upgrade electron from 8.5.2 to 8.5.3

Snyk has created this PR to upgrade electron from 8.5.2 to 8.5.3.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

gniezen committed:
Merge pull request #1343 from tidepool-org/snyk-upgrade-727320905fb1e91f0daa6975a2075128

[Snyk] Upgrade electron from 8.5.2 to 8.5.3

gniezen committed:
Merge branch 'master' into webmtp

gniezen committed:
remove test suffix

snyk-bot committed:
fix: upgrade electron-log from 4.2.4 to 4.3.0

Snyk has created this PR to upgrade electron-log from 4.2.4 to 4.3.0.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

gniezen committed:
Merge pull request #1344 from tidepool-org/snyk-upgrade-4958788c6bef58a126ad0071495e3dfe

[Snyk] Upgrade electron-log from 4.2.4 to 4.3.0

HannuHyttinen committed:
Bug report template, ver1.

HannuHyttinen committed:
Create Bug report

jlaunonen committed:
Merge pull request #29 from Sensotrend/localization-main

Main localization

HannuHyttinen committed:
Delete Bug report

HannuHyttinen committed:
Merge branch 'master' of https://github.com/Sensotrend/sensotrend-uploader

HannuHyttinen committed:
Tried to fix.

HannuHyttinen committed:
Create Bug report

HannuHyttinen committed:
Delete Bug report

HannuHyttinen committed:
Fix 2.

HannuHyttinen committed:
Merge branch 'master' of https://github.com/Sensotrend/sensotrend-uploader

HannuHyttinen committed:
Fix 3

jlaunonen committed:
Merge remote-tracking branch 'origin/main' into sensotrend-uploader

HannuHyttinen committed:
Fix 4

HannuHyttinen committed:
Improvement.

HannuHyttinen committed:
Improvement.

HannuHyttinen committed:
Part 2 improved.

HannuHyttinen committed:
A misspelling correction.

HannuHyttinen committed:
A terminology fix.

gniezen committed:
Merge branch 'master' into webmtp

gniezen committed:
Merge pull request #1284 from tidepool-org/webmtp

Upload DASH PDMs using WebMTP (UPLOAD-349)

HannuHyttinen committed:
Modification after discussion with team.

HannuHyttinen committed:
Clarifications.

HannuHyttinen committed:
System info clarified.

snyk-bot committed:
fix: upgrade @babel/runtime-corejs2 from 7.12.1 to 7.12.5

Snyk has created this PR to upgrade @babel/runtime-corejs2 from 7.12.1 to 7.12.5.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

gniezen committed:
Merge pull request #1347 from tidepool-org/snyk-upgrade-5a6f57af5fa0785410e2bc0250ac293e

[Snyk] Upgrade @babel/runtime-corejs2 from 7.12.1 to 7.12.5

HannuHyttinen committed:
Fixes and improvements.

gniezen committed:
Merge branch 'master' into multi-languages

gniezen committed:
v2.34.0-multi-languages.1

snyk-bot committed:
fix: upgrade multiple dependencies with Snyk

Snyk has created this PR to upgrade:
  - @babel/plugin-proposal-numeric-separator from 7.12.1 to 7.12.5.
    See this package in npm: 
  - @babel/preset-react from 7.12.1 to 7.12.5.
    See this package in npm: 

See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

HannuHyttinen committed:
Improvements after discussion in the demosession.

HannuHyttinen committed:
Bold texts corrected.

gniezen committed:
Merge pull request #1348 from tidepool-org/snyk-upgrade-4df0104d7d7f9630c662d052c1a49150

[Snyk] Upgrade: @babel/plugin-proposal-numeric-separator, @babel/preset-react

gniezen committed:
removing jest plugin from global eslintrc as it causes an issue with template literals; specified global in file instead

gniezen committed:
Merge branch 'master' into multi-languages

krystophv committed:
switch out eslint babel parser

per: https://babel.dev/blog/2020/07/13/the-state-of-babel-eslint#the-future

krystophv committed:
be specific about eslint local version

krystophv committed:
`npx yarn-deduplicate yarn.lock`

snyk-bot committed:
fix: upgrade eslint-config-airbnb from 18.2.0 to 18.2.1

Snyk has created this PR to upgrade eslint-config-airbnb from 18.2.0 to 18.2.1.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/tidepool/project/e075bd55-9c42-4a0a-8ead-dca415c301dc?utm_source=github&utm_medium=upgrade-pr

gniezen committed:
Merge pull request #1349 from tidepool-org/snyk-upgrade-c2cb4bbd96d3e6b508e8bf71f9ecba07

[Snyk] Upgrade eslint-config-airbnb from 18.2.0 to 18.2.1

krystophv committed:
Revert "removing jest plugin from global eslintrc as it causes an issue with template literals; specified global in file instead"

This reverts commit 81d525783ba41d250c835fed935b616494d2c961.

krystophv committed:
fully resolve yarn.lock conflicts

krystophv committed:
Merge branch 'eslint-parser' into multi-languages

krystophv committed:
Merge branch 'master' into multi-languages

HannuHyttinen committed:
OWASP link in issue templates corrected.

HannuHyttinen committed:
Issue templateds improved.

jlaunonen committed:
Merge remote-tracking branch 'origin/sensotrend-uploader-2.34.0' into sensotrend-uploader

jlaunonen committed:
Merge remote-tracking branch 'origin/master' into sensotrend-uploader

jlaunonen committed:
Merge remote-tracking branch 'jehernandez/multi-languages' into sensotrend-uploader

jlaunonen committed:
Update server addresses.

By default, production.

jlaunonen committed:
Add login error text to translation.

- Also fix one translation.

jlaunonen committed:
Remove "made possible by" for now.

jlaunonen committed:
Statically translate about dialog.

jlaunonen committed:
Adjust about-dialog disclaimer.

mrinnetmaki commented:
Päänäkymän "Yksityisyys ja käyttöehdot" -linkki vie vielä osoitteeseen https://www.tidepool.org/legal/.
Saisiko ennemmin vain "Käyttöehdot" ja osoittamaan osoitteeseen https://www.sensotrend.fi/connect/eula ?
Jos halutaan sekä yksityisyys että käyttöehdot (@HannuHyttinen?), niin "Yksityisyys"-linkin pitäisi osoittaa osoitteeseen https://www.sensotrend.fi/connect/privacy/.

Ja Tuki-linkki vie Tidepoolin sivulle, ei tuki@sensotrend.com-sähköpostiin.

mrinnetmaki commented:
Apua-valikon tuki-linkki voisi nyt osoittaa suomenkieliseen osoitteeseen tuki@sensotrend.com, kun sellainenkin on olemassa.
Ja tuokin olisi kiva, jos olisi mahdollista määritellä lokalisaation avulla.

Ja sama About-dialogiin.

mrinnetmaki commented:
Apua-valikon Tietosuoja-linkki osoittaa osoitteeseen https://connect.sensotrend.fi/privacy. Oikea osoite tällä hetkellä on https://www.sensotrend.fi/connect/privacy/.

mrinnetmaki commented:
About-dialogin teksti olisi mun mielestä parempi näin:

Sensotrend Uploader on luokan 1 lääkinnällinen laite.
Sovellus on vielä kehitysvaiheessa.

(Ja tuo jälkimmäinen lause punaisella.)

Voitaisiin linkata vaatimustenmukaisuuslomakkeeseen, mutta halutaanko? @HannuHyttinen?

mrinnetmaki commented:
Päänäkymän rekisteröidy -linkki osoittaa osoitteeseen https://connect.sensotrend.fi/view/signup.
Nykyään oikea osoite on https://www.sensotrend.fi/connect/.

mrinnetmaki commented:
Nuo linkit, jotka osoittavat osoitteeseen https://connect.sensotrend.fi/ voisi toki ratkaista myös toteuttamalla palvelinpuolelle oikeanlaiset uudelleenohjaukset. Mitä mieltä @jlaunonen ja @SamThunder?

mrinnetmaki commented on app/components/DeviceTimeModal.js:
```diff
@@ -104,9 +107,9 @@ export class DeviceTimeModal extends Component {
-            <a href='https://support.tidepool.org/hc/en-us/articles/360034136632' target='_blank'>Click to learn more about meters and device time.</a>
+            {i18n.t('* Changing your device time will not change any previous records.')}<br/>
+            {i18n.t('All future readings will be in {{timezone}}.', { timezone: timezone })}
+            <a href='https://support.tidepool.org/hc/en-us/articles/360034136632' target='_blank'>{i18n.t('Click to learn more about meters and device time.')}</a>
```
This still points to Tidepool, but I'd say that's fine.

mrinnetmaki commented on app/components/DeviceTimeModal.js:
```diff
@@ -74,21 +77,21 @@ export class DeviceTimeModal extends Component {
-          Automatically update time to<br/>
-          {sundial.formatInTimezone(serverTime, timezone, 'LT')}{footnote}, and upload
+          {i18n.t('Automatically update time to')}<br/>
+          {sundial.formatInTimezone(serverTime, timezone, 'LT')}{footnote}, {i18n.t('and upload')}
```
Do we parameterize sundial (or moment.js?) somewhere else?
If not, we could also modify the masks here based on localization.
See https://github.com/tidepool-org/sundial/blob/master/lib/datetimeWrapper.js#L262 for how sundial handles masks, and https://momentjs.com/docs/#/displaying/format/ for available masks.

mrinnetmaki replied:
(I did not test this yet, just reading through...)

mrinnetmaki replied:
moment seems to be localized, see https://github.com/Sensotrend/sensotrend-uploader/pull/34/files#diff-70b6a4ba131b90d3682f0b2ba111ccabb455e5584068bdfc9744f713f79db1a5R36

jlaunonen replied:
LT is locale-independent mask so it should not need any changes regardless of locale. Moment should know its locale, set in app/index.js:36. A problem arises however when a mask that is non-locale-independent is used without localization (e.g. app/components/TimezoneDropdown.js:123).

mrinnetmaki commented on app/constants/errors.js:
```diff
@@ -37,5 +37,6 @@ module.exports = {
-  E_DEXCOM_CONNECTION: 'Tidepool is having trouble connecting to your Dexcom receiver. Please make sure your other Dexcom uploading software programs are closed (like Dexcom Clarity or Glooko/Diasend). You can also try using another micro-USB cable. Some micro-USB cables are designed to carry a signal for power only.'
+  E_DEXCOM_CONNECTION: 'Tidepool is having trouble connecting to your Dexcom receiver. Please make sure your other Dexcom uploading software programs are closed (like Dexcom Clarity or Glooko/Diasend). You can also try using another micro-USB cable. Some micro-USB cables are designed to carry a signal for power only.',
+  E_USB_CABLE: 'Your device doesn\'t appear to be connected. You can try using another USB cable, as some USB cables are designed to carry a signal for power only.',
 };
```
The error messages don't seem to be localized yet. Correct?

jlaunonen replied:
Some are overridden with localized version on display. Some are not. e.g. E_OFFLINE is compared with string comparison and replaced with VersionCheckError.defaultProps.errorText.OFFLINE value in render().

mrinnetmaki replied:
:+1:

mrinnetmaki commented on app/containers/App.js:
```diff
@@ -67,31 +67,31 @@ const serverdata = {
     UPLOAD_URL: 'http://localhost:1300/tpupload',
     DATA_URL: 'http://localhost:1300/tpdata',
-    BLIP_URL: 'http://localhost:1300/'
+    BLIP_URL: 'http://localhost:1300/view'
```
I added a comment regarding the url's in the discussion of the PR.
Not sure whether we want to resolve things here or on server side with redirects.

mrinnetmaki commented on app/components/TimezoneDropdown.js:
```diff
@@ -111,15 +114,19 @@ class TimezoneDropdown extends React.Component {
       return (
         <div className={styles.timeDetail}>
-          {this.props.isClinicAccount ? 'The ' : 'Your '} device times should be approximately {sundial.formatInTimezone(new Date(), this.props.targetTimezone, 'h:mm a')}
+          {text} {sundial.formatInTimezone(new Date(), timez, 'h:mm a')}
```
The sundial time mask should be localized too, right? 'LT' instead of 'h:mm a'?

mrinnetmaki commented on app/main.dev.js:
```diff
@@ -292,32 +305,32 @@ operating system, as soon as possible.`,
+        label: i18n.t('Get Support'),
         click() {
-          shell.openExternal('http://support.tidepool.org/');
+          shell.openExternal('mailto:support@sensotrend.com');
```
It would be nice to get these localized too.
So we'd get tuki@sensotrend.com for Finnish and support@sensotrend.com for English versions...

mrinnetmaki commented on app/main.dev.js:
```diff
@@ -104,19 +114,22 @@ app.on('ready', async () => {
 While Windows 7 will continue to work, Microsoft recommends you
 start planning to upgrade to Windows 10, or an alternative
-operating system, as soon as possible.`,
+operating system, as soon as possible.`),
```
The Finnish localization for this did not seem to have all the line breaks in place.
This may distort the final dialog?

(I'm just assuming the line breaks in this code have a reason to exist, other than keeping the line length of the source code short?)

jlaunonen replied:
Seems the dialog is a bit more wide this way, but it wraps the text automatically. So I'd say it doesn't matter. (One line for topic, two lines for body text.)

mrinnetmaki replied:
:+1:

mrinnetmaki commented on app/main.dev.js:
```diff
@@ -170,120 +183,120 @@ operating system, as soon as possible.`,
         type: 'separator'
       }, {
-        label: 'Hide Sensotrend Uploader',
+        label: i18n.t('Hide Tidepool Uploader'),
```
I have mixed feelings about how the app menus are translated.
I'm using an English locale on my computer, but do appreciate apps localized in Finnish.
However, my expectation would have been that everything "closer to the OS" would still be in the language of the operating system.

Just noting my feeling, I don't have a strong opinion on what should be the right approach here.

jlaunonen replied:
I'd say we may have accustomed to have OS in certain language while using some web-service in another. I can't remember any application (aside from browsers when they display third party content such as web sites) that have different parts of the UI messages localized differently (so not counting dates and such). Even if this app uses a browser to render part of the UI I'd say the case isn't same as is with browsers.

mrinnetmaki replied:
:+1:

mrinnetmaki commented on build-all.sh:
```diff
@@ -1,6 +1,6 @@
+sed -i '' 's/uploads.tidepool.org/connect.sensotrend.fi\/tpupload/g' node_modules/tidepool-platform-client/tidepool.js
+sed -i '' 's/data.tidepool.org/connect.sensotrend.fi\/tpdata/g' node_modules/tidepool-platform-client/tidepool.js
 
 yarn package-all
```
By the way, I don't think we really need these build scripts.
The tidepool-platform-client seems to be configured OK by just the configs and the presets in App.js

mrinnetmaki commented on electron-builder-publish.js:
```diff
@@ -0,0 +1,125 @@
+    publisherName: [
+      'Sensotrend Oy'
+    ],
+    rfc3161TimeStampServer: 'http://timestamp.digicert.com'
```
Note to self, we need to add the code signing certificate info here...

mrinnetmaki commented on locales/fi/translation.json:
```diff
@@ -0,0 +1,231 @@
+  "All future readings will be in {{timezone}}.": "Kaikki tulevat lukemat tulevat olemaan aikavyöhykkeellä {{timezone}}.",
+  "Click to learn more about meters and device time.": "Klikkaa lukeaksesi lisää mittareista ja laitteen ajasta.",
+  "Remember to tap \"Export\" on the PDM before clicking \"Upload\".": "Muista valita PDM:stä \"Export\" ennen kuin klikkaat \"Lähetä\".",
+  "Your {{text}} doesn't appear to be in": "{{text}} ei näytä olevan",
```
How about 
"Laitteesi aika ei näytä olevan aikavyöhykkeellä", "Liitetty {{text}} ei näytä olevan ajassa",  or something similar and more descriptive?

mrinnetmaki commented on locales/fi/translation.json:
```diff
@@ -0,0 +1,231 @@
+  "Privacy and Terms of Use": "Yksityisyys ja käyttöehdot",
+  "Loading...": "Ladataan...",
+  "Upload in progress!\nPlease wait to change device selection.": "Lähetys on kesken!\nOle hyvä ja odota ennen laitevalinnan vaihtamista.",
+  "Set up data storage to upload devices.": "Määrittele tietovaranto lähettääksesi laitteet.",
```
Tietojen tallennuspaikka?

mrinnetmaki commented on locales/fi/translation.json:
```diff
@@ -0,0 +1,231 @@
+  "Set up data storage to upload devices.": "Määrittele tietovaranto lähettääksesi laitteet.",
+  "Upload in progress!\nPlease wait to log out.": "Lähetys kesken!\nOle hyvä ja odota ennen uloskirjautumista.",
+  "Set up data storage": "Määrittele tietovaranto",
+  "Or, ask the person you are uploading for to grant you access to upload.": "Tai pyydä henkilöä jonka tietoja lähetät antamaan pääsy lähetykseen.",
```
"Tai pyydä henkilöä, jonka tietoja lähetät, antamaan lupa tietojen lähetykseen"

mrinnetmaki commented on locales/fi/translation.json:
```diff
@@ -0,0 +1,231 @@
+  "You need to be logged in to update your preferences.": "Sinun on oltava kirjautuneena sisään päivittääksesi valintojasi.",
+  "This email is already associated with a Tidepool account.": "Tämä sähköpostiosoite on jo yhdistetty Sensotrend-tiliin.",
+  "We can't save your device and timezone selection right now.": "Emme voi tallettaa laite- ja aikavyöhykevalintaasi juuri nyt.",
+  "Please update to a modern operating system": "Voisitko päivittää käyttöjärjestelmäsi",
```
Kysymysmerkki perään?

mrinnetmaki commented on scripts/about.html:
```diff
@@ -110,13 +110,13 @@
     <span class="alert">
-    This demo version does not meet all regulatory requirements.<br/>
-    Not for clinical use.
+    Sovellus on vielä kehitysvaiheessa.<br/>
```
Laitoin tästä kommentin PR:n discussion-osioon...

mrinnetmaki commented on scripts/about.html:
```diff
@@ -125,10 +125,10 @@ <h4>Based on <a href="https://github.com/tidepool-org/uploader">Tidepool Univers
     </div>
     <div class="copyrights">${:copyrights}
-<br/>Distributed under ${license}</div>
+<br/>Jaettu ${license} lisenssin alaisena</div>
```
Jaettu ${license} -lisenssin alaisena

?

mrinnetmaki replied:
Tai  "Jaettu lisenssillä ${license}" ?

mrinnetmaki reviewed (commented):
Added comments based on reading the code. Still haven't tested the version properly, but wanted to provide feedback, mainly to get the discussion going on how to handle the external URLs...

jlaunonen commented:
Noi osotteet missä on /view/ mukana derivoidaan blip_urlista eli ovat ympäristökohtaisia, joten pitää tunkata jotain jos nuo tietyt haluaa toisin. Tuossahan on myös forgotten password linkki, joka on `$blip_url/request-password-from-uploader`.
Privacy ja Eula ok, ne voi laittaa erillisiksi linkeiksi.

jlaunonen committed:
Replace one legal link with two in footer.

jlaunonen committed:
Localization fixes.

jlaunonen committed:
Also statically-localize support address in about dialog.

mrinnetmaki commented:
Mac puhuu vielä valikossa Tidepool Uploaderista.

![image](https://user-images.githubusercontent.com/5329276/102066608-b2780980-3e02-11eb-9eed-9b74d518e3c1.png)

mrinnetmaki commented:
Macillä about-dialogi on tyhjä.

jlaunonen commented:
> Mac puhuu vielä valikossa Tidepool Uploaderista.

Ei pitäis puhua, edes englanniksi. Espanjaa ei ole tunkattu, joten se puhuis. Tonhan ei edes pitäis olla englanniksi config.i18next.js:ssä olevan lng:'fi' asetuksen takia.

mrinnetmaki commented:
Joo, oon varmaan buildannut jotenkin väärin...

HannuHyttinen commented:
> Macillä about-dialogi on tyhjä.

Samoin Windowsilla.

HannuHyttinen commented:
> About-dialogin teksti olisi mun mielestä parempi näin:
> 
> Sensotrend Uploader on luokan 1 lääkinnällinen laite.
> Sovellus on vielä kehitysvaiheessa.
> 
> (Ja tuo jälkimmäinen lause punaisella.)
> 
> Voitaisiin linkata vaatimustenmukaisuuslomakkeeseen, mutta halutaanko? @HannuHyttinen?

Mikaelin ehdottama teksti kuulostaa hyvältä.
Ei minusta tarvitse linkata vaatimuksenmukaisuuslomakkeeseen.

mrinnetmaki commented on app/components/Footer.js:
```diff
@@ -46,11 +46,14 @@ export default class Footer extends Component {
-          <div>
-            <a className={styles.footerLink} href="http://support.tidepool.org/" target="_blank">{i18n.t('Get Support')}</a>
+          <div className={styles.el1}>
+            <a className={styles.footerLink} href={i18n.t('mailto:support@sensotrend.com')} target="_blank">{i18n.t('Get Support')}</a>
```
👍

mrinnetmaki commented on app/components/Footer.js:
```diff
@@ -46,11 +46,14 @@ export default class Footer extends Component {
-          <div>
-            <a className={styles.footerLink} href="http://tidepool.org/legal/" target="_blank">{i18n.t('Privacy and Terms of Use')}</a>
+          <div className={styles.el2}>
+            <a className={styles.footerLink} href="https://www.sensotrend.fi/connect/privacy/" target="_blank">{i18n.t('Privacy')}</a>
```
Nämä voisi halutessaan laittaa toimimaan sen BLIP_URLin kautta, mutta olkoot nyt noin toistaiseksi.

mrinnetmaki commented on app/components/TimezoneDropdown.js:
```diff
@@ -120,7 +120,7 @@ class TimezoneDropdown extends React.Component {
       return (
         <div className={styles.timeDetail}>
-          {text} {sundial.formatInTimezone(new Date(), timez, 'h:mm a')}
+          {text} {sundial.formatInTimezone(new Date(), timez, 'LT')}
```
Tämänhän voisi myös lähettää Tidepoolille, ihan pikkuisena erillisenä pull requestina. :)

mrinnetmaki replied:
Ihan siihen alkuperäiseen toteutukseen liittyen.

jlaunonen replied:
Joo. Tohon lokalisointiin liittyy joitakin muitakin paikkoja (ainakin lisää laitelistaa) mitkä pitäis viedä sinne jollain aikataululla...

mrinnetmaki commented on locales/fi/translation.json:
```diff
@@ -68,16 +68,18 @@
-  "Privacy and Terms of Use": "Yksityisyys ja käyttöehdot",
+  "Privacy": "Yksityisyys",
+  "Terms of Use": "Käyttöehdot",
+  "mailto:support@sensotrend.com": "mailto:tuki@sensotrend.com",
```
:+1:

jlaunonen committed:
Fix about-dialog related missing files from package builds.

jlaunonen committed:
Make possible to close about-dialog with escape.

More dialog-like usability.

jlaunonen committed:
Update about-dialog disclaimer once more.

jlaunonen committed:
Enhance esc handling.

Require no modifiers were used and that we are not between key
composition. Also require, that the sole esc has been pressed down
before handling up.

jlaunonen committed:
Make about-dialog address text selectable (indirectly via .inline).

Expand .inline slightly so that the selection drag can be started
outside of the immediate element.

jlaunonen committed:
Update version information to 2.34.2.
